src/d3a/d3a_core/export.py

```diff
@@ -132,6 +131,11 @@ def export(self):
                 ConstSettings.GeneralSettings.SUPPLY_DEMAND_PLOTS:
             self.plot_supply_demand_curve(self.area, self.plot_dir)
         self.move_root_plot_folder()
+        self.export_json_data(self.directory)
+
+    def data_to_csv(self, area, is_first):
+        with page_lock:
```

Comment:
Is the page_lock really necessary? Doesnt `with open() as file` deals with that?

Reply:
Yes, you are right. Removing it!